common/buildings/00_terrain_specific_buildings.txt

```diff
@@ -292,6 +292,7 @@ pastures_07 = {
 	}
```

Comment:
Check the modifiers again. You missed `archers_damage_mult` in `ramparts_05`.

Reply:
Fixed, regimental ground 1/2 are supposed to not have massive_aerial modifiers, I feel like that would be to op to fast

Reply:
`heavy_infantry_toughness_mult` in `blackrock_depths`.

Reply:
`heavy_infantry_damage_mult` in `holmslatr`.